cdap-common/bin/service

```diff
@@ -208,6 +210,7 @@ run() {
   # Setup classpaths.
   cdap_set_classpath "${COMPONENT_HOME}" "${CDAP_CONF}"
   cdap_set_hive_classpath || exit 1
+  cdap_set_spark
```

Comment:
my thinking was whether we are launching a cdap service or a running a utility class, the environment should be consistent, and therefore we should log warnings consistently.